app/test_main.py

```diff
@@ -1,3 +1,45 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestPetsHumenAge:
+    def test_should_params_be_int_not_str(self) -> None:
+        with pytest.raises(TypeError):
+            get_human_age("15", "15"), "parameters must be an integer, not str"
+
+    def test_should_convert_negative_years_to_0(self) -> None:
+        assert get_human_age(-5, -15) == [0, 0], \
+            "negative years should convert into 0 human age."
+
+    def test_should_convert_0_to_0(self) -> None:
+        assert get_human_age(0, 0) == [0, 0], \
+            "0 years should convert into 0 human age."
+
+    def test_should_convert_before_15_year_to_0(self) -> None:
+        assert get_human_age(5, 10) == [0, 0], \
+            "1-14 years should convert into 0 human age."
+
+    def test_should_convert_15_year_to_1(self) -> None:
+        assert get_human_age(15, 15) == [1, 1], \
+            "15 years should convert into 1 human age."
+
+    def test_should_convert_between_15_and_24_year_to_1(self) -> None:
+        assert get_human_age(18, 23) == [1, 1], \
```

Comment:
better to test edged variant 23 for both: cat and dog

Reply:
Ok, corrected